bb-new.html

```diff
@@ -0,0 +1,304 @@
+                  </div>
+              </div>
+              <div class="dropdown">
+                <a href="#industries">Industries</a>
+                  <div class="dropdown-content">
+                    <a href="#financial">Financial Services</a>
+                    <a href="#healthcare">Healthcare</a></li>
+                    <a href="#professional">Professional Services</a>
+                    <a href="#education">Education</a>
+                    <a href="#government">Government</a>
+                    <a href="#utilties">Utilities & Manufactoring</a>
+                    <a href="#transporation">Transportation</a>
+                    <a href="#consumer">Consumer industries</a>
+                  </div>
+              </div>
+              <div class="dropdown">
+                <a href="#partners">Partners</a>
+                  <div class="dropdown-content">
+                    <a href="#financial">Financial Services</a>
+                    <a href="#healthcare">Healthcare</a></li>
+                    <a href="#professional">Professional Services</a>
+                    <a href="#education">Education</a>
+                    <a href="#government">Government</a>
+                    <a href="#utilties">Utilities & Manufactoring</a>
+                    <a href="#transporation">Transportation</a>
+                    <a href="#consumer">Consumer industries</a>
+                  </div>
+              </div>
+              <div class="dropdown">
+                <a href="#partners">Support</a>
+                  <div class="dropdown-content">
+                    <a href="#products">Support by Products</a>
+                    <a href="#smartphone">Smartphones & Apps</a></li>
+                    <a href="#information">Products Inforamtion</a>
+                  </div>
+              </div>
+              <div class="dropdown">
+                <a href="#blog">Blog</a>
+                  <div class="dropdown-content">
+                    <a href="#enterprise">Enterprise Blog</a>
+                    <a href="#developer">Developer</a></li>
+                    <a href="#support">Support Blog</a>
+                  </div>
+              </div>              
+              <div class="dropdown topnav-right">
+                <a href="#contact">Telephone icon</a>
+                  <div class="dropdown-content">
+                    <a href="#register">Register for updates</a>
+                    <a href="#sales">Contact Sales</a></li>
+                    <a href="#contactus">Contact Us</a>
+                  </div>
+                </div> 
+              <div class="topnav-right">
+                <a href="#search">Search icon</a>
+              </div>           
+            </div>
+          </header>         
+        </div>         
+      </div>
+      <div class="row">
+        <div class="column">
+            <img src="./images/windmill.jpg" alt="several windmills in a line">
```

Comment:
Do not directly add images this way when they are meant to span an entire div.
Instead of adding an img and then absolute positioning everything within the div, you instead use the `background-image` property on a class added to the div on line 99 and then the items within that div, will automatically "go on top" of the image.

Look up the use of background-image.